src/pages/patterns/notification-pattern/index.mdx

```diff
@@ -0,0 +1,307 @@
+---
+title: Notifications
+description: Notifications are an important method of communicating with users and providing feedback.
+---
+
+<PageDescription>
+
+Notifications are an important method of communicating with users and providing feedback. They range from granular, inline responses to user actions to system-wide messages.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Designing with notifications</AnchorLink>
+<AnchorLink>Notification status</AnchorLink>
+<AnchorLink>Notification types</AnchorLink>
+<AnchorLink>Considerations when designing</AnchorLink>
+<AnchorLink>Accessibility</AnchorLink>
+<AnchorLink>Related components and patterns</AnchorLink>
+<AnchorLink>References</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+</AnchorLinks>
+
+## Overview
+
+It’s important to keep users informed and send notifications when there is an update or status change they should be aware of. Users should always be given appropriate and timely messages to help them understand whether they are moving towards their goal. Ensure your notifications are relevant, timely, and informative.
+
+| Key principle | Definition                                                                                              |
+| ------------- | ------------------------------------------------------------------------------------------------------- |
+| Relevant      | Notifications should be related to the user's goals and presented in the context of what they are doing |
+| Timely        | Users should be kept up-to-date with prompt notifications and ensure they see critical immediately      |
+| Informative   | Provide users with the context and next-steps needed to understand and address the notification         |
+
+Be considerate of your users when sending a notification. Notifications that are too frequent or disruptive create negative experiences and drive users from platforms.
+
+Notifications are comprised of status and type. Their status signifies the purpose of the information being conveyed. Notification types allow you to tailor the disruptiveness of the notification to the specific situation. Notification status and type options in Carbon should be combined to create notifications that are relevant, timely, and informative for each use case.
+
+<Row>
+<Column colSm={2} colMd={2} colLg={3}>
```

Comment:
```suggestion
<Column colSm={2} colMd={2} colLg={4}>
```

The text in the titles is wrapping at the large breakpoint (looks fine on mobile/tablet) so might work better as a 4 column. Not a huge deal either way though.